owncloudApp/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -692,7 +693,7 @@ void loadStateFrom(Bundle savedInstanceState) {
     }
 
     private void clearLocalSearchView() {
-        ((FileActivity) requireActivity()).hideSoftKeyboard();
+        FragmentExtKt.hideSoftKeyboard(this);
```

Comment:
You could use the ActivityExtension here and save that FragmentExtension if it is only used here.

```suggestion
        ActivityExtKt.hideSoftKeyboard(requireActivity());
```